pages/tools/starknet-js.mdx

```diff
@@ -0,0 +1,23 @@
+# Starknet.js Overview
+[Starknet.js](https://www.starknetjs.com/) is a lightweight JavaScript/TypeScript library enabling interaction between your DApp and Starknet Inspired by ethers.js, it offers a familiar API and allows you to interact with Starknet accounts, providers, and contracts. Key functionalities include account management, network interaction, contract interfacing, and deployment automation.
+
+## Installing Starknet.js
+Follow the official Starknet.js installation instructions: https://www.starknetjs.com/docs/guides/intro
+
+## Use Starknet.js in Madara
+Starknet.js provides a high-level abstraction, streamlining common tasks like contract management, account interactions, and transaction processing.
+
+### Configuring Starknet.js for Madara
+Starknet.js functions as a layer between your dApp and the Madara network. It creates providers that abstract contract deployments, account management, and transaction execution. 
+
+![stack](../../static/img/starknet-js.png)
+
+Madara offers predeployed accounts loaded with test tokens for development and testing. Use them directly with contracts or fund your Argent X/Braavos wallet. 
```

Comment:
we can specify that if we run if `--dev` mode, all predeployed accounts with their pks are loggged. maybe then you can also show the example code for faucet.js